tools/clang/lib/SPIRV/SpirvEmitter.h

```diff
@@ -1216,6 +1224,15 @@ class SpirvEmitter : public ASTConsumer {
   bool spirvToolsTrimCapabilities(std::vector<uint32_t> *mod,
                                   std::string *messages);
 
+  // \brief Runs the upgrade memory model pass using SPIRV-Tools's optimizer.
+  // This pass will modify the module so that it conforms to the Vulkan memory
+  // model instead of the pass. Removes unused capabilities from the given
+  // SPIR-V module |mod|, and returns info/warning/error messages via
+  // |messages|. This pass doesn't trim all capabilities. To see the list of
+  // supported capabilities, check the pass headers.
```

Comment:
```suggestion
```

This is from the pass above, but doesn't apply to this one no?

Reply:
Fixed.